### ISSUE TITLE
Ability to set search modal placeholder text color

### ISSUE THREAD

**Issue by quantumleap33** [tomusborne/generatepress#544]
We need to be able to set the search modal's placeholder text color from the Customizer. When field background color is a lighter color, the current placeholder text `Search...` is not visibly clear.

**Comment by tomusborne** [comment]
What browser are you using? Firefox seems to use the text color but fades it a little.

**Comment by quantumleap33** [comment]
Yeah, the issue is when using a kind of grey background field background GP makes a decision on the placeholder that is not ideal. Is it difficult to have an extra option for the placeholder?
https://vimeo.com/832457338/ef9344346d
